sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client_connection_async.py

```diff
@@ -2384,7 +2384,10 @@ def __GetBodiesFromQueryResult(result):
                 return __GetBodiesFromQueryResult(results)
 
         result, self.last_response_headers = await self.__Post(path, request_params, query, req_headers, **kwargs)
-
+        if http_constants.HttpHeaders.IndexUtilization in self.last_response_headers:
+            INDEX_METRICS_HEADER = http_constants.HttpHeaders.IndexUtilization
+            index_metrics_raw = self.last_response_headers[INDEX_METRICS_HEADER]
```

Comment:
same as the other one
```suggestion
        if self.last_response_headers.get(http_constants.HttpHeaders.IndexUtilization) is not None:
            index_metrics_raw = self.last_response_headers[http_constants.HttpHeaders.IndexUtilization]
```